composer.json

```diff
@@ -120,7 +120,7 @@
         {
             "_comment": "Amitai's fork is the main development repository for Organic Groups. Remove this once a D8 release is made on drupal.org.",
             "type": "vcs",
-            "url": "git@github.com:amitaibu/og"
+            "url": "git@github.com:ec-europa/og"
```

Comment:
Comment changed. One of the patches included, solve temporarily the issue that gets it blocked by ISAICP-2311 which is the delete functionality. Can't we use this for now?
After all, the patch takes care of behat's incapability of cleaning the memberships because of the lack of functionality currently being built on 2311.
Anyway, bottom line, in order to unblock, I altered the method that cleans the collections after the behat scenarios to also delete the memberships at CollectionSubContext:cleanCollections();.
Is this acceptable until 2311 is in place?